### PR TITLE
Update SecurityAdvisoryHandling.md based on feedback

### PR DESCRIPTION
**Pull request by CodeMonkeyLeet**
Incorporate comments from @markg-intel in #3556

Signed-off-by: Simon Leet <simonle@microsoft.com>

### CODE REVIEW

**Review by markg-intel — APPROVED** [review]
Looks good to me. Thanks for incorporating @CodeMonkeyLeet

File: docs/Community/SecurityAdvisoryHandling.md [hunk 1]
```diff
@@ -74,10 +75,16 @@ The Open Enclave SDK project currently relies on the principles for [Coordinated
...
+
+While the primary goal of CVD is to avoid disclosure before a fix is released, the OE SDK security reviewers will also work to
+accommodate the reporter's preferences on the timing for disclosing and fixing the vulnerability. For example, if a researcher
+believes that releasing the fix far ahead of their disclosure publication or presentation diminishes the impact of their work, the
+security reviewers will work with them to determine the best course of action, such as releasing the fix on the same day as their
+planned disclosure event.
 
 > - [ ] TODO: Establish a process and acceptance criteria for registering with OE SDK as a _Trusted Stakeholder_.
->   - A Trusted Stakeholder may not be a committer or maintainer of the SDK, but has a vested interest in the security of the OE SDK.
+>   - A Trusted Stakeholder may not be a committer or maintainer of the SDK but has a vested interest in the security of the OE SDK.
```
Comment (dthaler) [inline docs/Community/SecurityAdvisoryHandling.md:87]:
```suggestion
>   - A Trusted Stakeholder need not be a committer or maintainer of the SDK but can be anyone with a vested interest in the security of the OE SDK.
```

**Review by dthaler — APPROVED** [review]

File: docs/Community/SecurityAdvisoryHandling.md (hunk 1 above)
Comment (CodeMonkeyLeet) [reply docs/Community/SecurityAdvisoryHandling.md]: Updated as suggested.

**Pull request merged**